docs/start/index.html

```diff
@@ -119,8 +119,12 @@ <h2 id="prerequisites">Prerequisites</h2>
   <!-- <li><a href="openshift.html">Red Hat OpenShift</a> or <a href="okd.html">OKD</a></li> -->
 </ul>
 
-<p>These instructions require <code>kubectl</code> version 1.15 or later.  See the
-<a href="https://kubernetes.io/docs/tasks/tools/install-kubectl/">kubectl installation
+<p>These instructions require the proper version of <code>kubectl</code> for your
+installation.  For Minikube installations see <a href="minikube.html#prerequisites">Minikube
+Prerequisites</a>.  Other installations
+require <code>kubectl</code> version 1.15 or later.</p>
```

Comment:
What does other installations refer to here?  Is the idea that Minikube needs something special?  I haven't run into that.  The idea with '1.15' or later is that the instructions use 'minikube config set-context *--current*', which was new with 1.15.